### DISCUSSION TITLE
W2UI 1.5 availability via a secure CDN

### DISCUSSION THREAD

**Discussion by STTamhane** [vitmalina/w2ui#2050] [Q&A]
Hi, I currently use W2UI via Cloudflare CDN from behind a corporate firewall. I like the features and am even more impressed with those in release 1.5. However, I do not see a secure CDN that distributes this version of min.js and min.css. Cloudflare does have kickstart 1.5, but we are not using kickstart. Are there any plans to distribute release 1.5 js and css via secure CDN? We cannot connect to urls with http://; only with https://

Thanks.

**Comment by vitmalina** [comment]
No plans. The lib is very small, you can put it on your server. There are too many cdns and distribution channels and I gave up publishing changes to them all. I publish to NPM, but besides that no where else. Please download from w2ui.com and use it within your project. `nmp install w2ui` should work for you.

As for KickStart, it is deprecated in favor of https://github.com/vitmalina/w2ui-starter-es6

**Comment by STTamhane** [comment]
Thanks. Will do.
